ert/data/_record.py

```diff
@@ -131,34 +137,71 @@ class BlobRecord(Record):
     data: blob_record_data
 
 
-class RecordCollection(_DataElement):
-    records: Union[Tuple[NumericalRecord, ...], Tuple[BlobRecord, ...]]
-    ensemble_size: Optional[int] = None
+_RecordTupleType = Union[Tuple[NumericalRecord, ...], Tuple[BlobRecord, ...]]
+
+
+class RecordCollection:
+    def __init__(self, records: Any, ensemble_size: Optional[PositiveInt] = None):
+        self._records: _RecordTupleType
+        try:
+            # Passing a single records indicates that a uniform record is created.
```

Comment:
So a record collection for a ensemble with one member is uniform? Even though it would not have been if the ensemble was larger?

Reply:
That comment looks like leftover, it is incorrect, will fix.